### ISSUE TITLE
tqdm.keras add keyword arguments

### ISSUE THREAD

**Issue by Stroper111** [tqdm/tqdm#1065]
- [X] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [ ] visual output bug
    + [ ] documentation request (i.e. "X is missing from the documentation." If instead I want to ask "how to use X?" I understand [StackOverflow#tqdm] is more appropriate)
    + [X] new feature request
- [X] I have visited the [source website], and in particular
  read the [known issues]
- [X] I have searched through the [issue tracker] for duplicates
- [X] I have mentioned version numbers, operating system and
  environment, where applicable:
```python
>>> import tqdm, sys
>>> print(tqdm.__version__, sys.version, sys.platform)
>>> 4.51.0 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)] win32
```

When using `tqdm.keras.TqdmCallback` it is currently not possible to set normal  `tqdm` features, such ass `desc`. 

I am not sure if it was intended this way, but a work around would be to allow `**kwargs` to be processed by the  `tqdm_class`, this would make  the following possible:

```python
tqdm.keras.TqdmCallback(epochs=20, verbose=0,  desc='Training'.ljust(20))
Training            : 100%|██████████| 20/20 [00:20<00:00,  1.01s/epoch, loss=0.169, accuracy=0.661]
```

[source website]: https://github.com/tqdm/tqdm/
[known issues]: https://github.com/tqdm/tqdm/#faq-and-known-issues
[issue tracker]: https://github.com/tqdm/tqdm/issues?q=
[StackOverflow#tqdm]: https://stackoverflow.com/questions/tagged/tqdm

**Comment by casperdcl** [comment]
Will be fixed by #1059

**Comment by Stroper111** [comment]
Thanks! Love the library and layout and this is great for using it together with Keras.
